### ISSUE TITLE
BUG: Data Entry fields cropping data

### ISSUE THREAD

**Issue by dwg1957** [TheInfiniteKind/moneydance_open#59]
Fields, and especially noticeable on date fields is that an entry box is created when you enter or edit data and the box is considerably smaller than the data causing values to be cropped.

**Comment by dtdseeker** [comment]
Probably pivotal to the next "preview" release. Still need to figure out what is a bug fix release vs. a "look and see the bugs" release.

**Comment by sreilly** [comment]
@dwg1957 could you please post a screenshot?

**Comment by dwg1957** [comment]
@sreilly 

Sure. The date of this transaction I have placed in edit mode is 23/4/2021. You can see the whole date is shown in the registry entries, but when in edit mode, the font is being changed and a smaller edit box is being created that crops the date.

![Cropping](https://user-images.githubusercontent.com/59605513/120170191-12912980-c244-11eb-8845-f0cd06042820.png)

**Comment by yogi1967** [comment]
Hmm. On Mac the edit window is smaller and it seems to truncate the left, but not shrink the text. So perhaps different issue(s) on different platforms?

**Comment by dwg1957** [comment]
My screen shot is from Windows with the default font. I can have a look at Linux if that is needed.

**Comment by dwg1957** [comment]
This looks like it could be a duplicate of #44

**Comment by sreilly** [comment]
I think this is probably a different issue to 44.  @dwg1957 can you confirm the build of moneydance that was running for this screenshot?

**Comment by dwg1957** [comment]
Fair enough. 

Sure it is build 3069

**Comment by yogi1967** [comment]
Latest build 3069.. Same(ish) issue on all platforms. Attached Screenshots. I created new datasets, deleted config.dict, and then entered new txns. Thus all default fonts and sizes.

Windows and Linux has quite a large left chop of the date field..

Similar on Mac, but 'looks' different and less dramatic, probably due to different LAF. I noticed that also when a line is selected, that the date also gets truncated??

<img width="802" alt="Mac" src="https://user-images.githubusercontent.com/22958485/120315618-72cfac00-c2d4-11eb-8481-61efc16f9e88.png">
![Windows](https://user-images.githubusercontent.com/22958485/120315623-73684280-c2d4-11eb-887c-19f867b9e463.png)
![Linux](https://user-images.githubusercontent.com/22958485/120315624-7400d900-c2d4-11eb-80ef-00d28896a42d.png)
